python/resdata/summary/rd_cmp.py

```diff
@@ -95,7 +90,7 @@ def end_time_equal(self):
         return self.test_case.endTimeEqual(self.ref_case)
 
     def cmp_summary_vector(self, key, sample=100):
-        """Will compare the summary vectors according to @key.
+        r"""Will compare the summary vectors according to @key.
```

Comment:
This seems non standard. The problem is the use of `\int` as the integral sign in the string (from tex). I suggest changing `\int` to either `\\int` or ∫ (U+222B)